userbenchmark/triton/run.py

```diff
@@ -139,8 +146,12 @@ def get_parser(args=None):
     args, extra_args = parser.parse_known_args(args)
     if args.op and args.ci:
         parser.error("cannot specify operator when in CI mode")
-    elif not args.op and not args.ci:
-        parser.error("must specify operator when not in CI mode")
+    elif not args.op and not args.ci and not args.op_collection:
```

Comment:
since we set a default value for --op-collection, this will never be triggerred.